presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationCompiler.java

```diff
@@ -119,6 +124,13 @@ public static BindableAggregationFunction generateAggregationBindableFunction(Cl
         return builder.build();
     }
 
+    private static List<LongVariableConstraint> parseLongVariableConstraints(Method inputFunction)
```

Comment:
I've added tests for that to last commit of PR which contains testing scenarios for whole annotation engine.